### ISSUE TITLE
Source maps no longer work in Emotion 10

### ISSUE THREAD

**Issue by lumio** [emotion-js/emotion#1076]
- `emotion` version: ^10.0.2
- `react` version: ^16.6.3

**Relevant code:**

```javascript
import React, { Component } from 'react';
// logo import

import styled from '@emotion/styled';

// The following won't generate correct source maps (with or without using macros)
const AppStyles = styled.div`
  text-align: center;

  .App-logo {
    animation: App-logo-spin infinite 20s linear;
    height: 40vmin;
  }
`

// rest of the default App.js from create-react-app
```

**What you did:**

1. Bootstrapped a new app with `create-react-app`. Tried both the macro version and the babel-plugin by ejecting the project
2. Converted the App.css into a styled component with `@emotion/styled`
3. Checked if it works... labels do indeed work 👍 source maps do not 👎 

**What happened:**

Source maps won't work with emotion 10. Downgrading to `emotion@^9`, `react-emotion@^9` and `babel-plugin-emotion@^9` will sort of "fix" it.

However, I know that `babel-plugin-emotion@^10` does something as it adds labels.
Additionally the `css` syntax seems to work and generates a source map. At least in combination with `Global`.

The following codesandbox uses macros, but the result is the same as described below.
https://codesandbox.io/s/moljl313kx

**Reproduction:**
See "What you did"

1. `create-react-app my-app`
2. `cd my-app`
3. `yarn add @emotion/core @emotion/styled @emotion/css babel-plugin-emotion`
4. `git commit -a 'stuff'`
5. `yarn eject`
6. Alter webpack.config.dev.js and add `babel-plugin-emotion` to the `babel-loader` options  
```js
          {
            test: /\.(js|mjs|jsx|ts|tsx)$/,
            include: paths.appSrc,
            loader: require.resolve('babel-loader'),
            options: {
              // create-react-app stuff

              plugins: [
                //
                // Basically this block down here:
                // You could simplify and just write 'emotion' without any options.
                //
                [
                  require.resolve('babel-plugin-emotion'), // a simple 'emotion' would do it as well
                  {
                    sourceMap: true,
                    autoLabel: true,
                    labelFormat: '[filename]--[local]',
                    cssPropOptimization: true,
                  },
                ],
                [
                  require.resolve('babel-plugin-named-asset-import'),
                  {
                    loaderMap: {
                      svg: {
                        ReactComponent: '@svgr/webpack?-prettier,-svgo![path]',
                      },
                    },
                  },
                ],
              ],
              // more create-react-app stuff
            },
          },
```

**Problem description:**
**tl;dr:** Source maps don't work in emotion 10 any longer.

**Suggested solution:**
Either downgrade to version 9 as suggested by [amaidah here](https://github.com/emotion-js/emotion/issues/844#issuecomment-438546602) or live without full source map support for now. Labels however seem to work straight away, so I would still use emotion 10.

If you want to downgrade to version 9, these packages work for me:
```
"babel-plugin-emotion": "^9.2.11",
"emotion": "^9.2.12",
"react-emotion": "^9.0.0"
```

A shoutout to the contributers and the author: emotion 10 looks awesome and clean! 👍

**Comment by lumio** [comment]
Thanks for the quick response @mitchellhamilton 
Sadly your PR is not fixing the problem, seen here: https://codesandbox.io/s/7jxqn8jj11

Also tested by upgrading @emotion/core, @emotion/styled and babel-plugin-emotion in my project to 10.0.3

**Comment by lumio** [comment]
Awesome! Thank you so much! Just tested it, and it works like a charm!

A side question... would it make sense to add a prepublish script that would build the packages?

**Comment by Andarist** [comment]
IMHO `prepare` is a better choice for build scripts (although its merits are not applicable to monorepos 😢 )

I would definitely be for adding a script for that.
